lib/widgets/message_list.dart

```diff
@@ -1417,10 +1430,10 @@ class MessageWithPossibleSender extends StatelessWidget {
                         letterSpacing: proportionalLetterSpacing(
                           context, 0.05, baseFontSize: 12))),
                 ])),
-              SizedBox(width: 16,
-                child: message.flags.contains(MessageFlag.starred)
-                  ? Icon(ZulipIcons.star_filled, size: 16, color: designVariables.star)
-                  : null),
+              SizedBox(
+                width: 16,
+                child: star,
+              ),
```

Comment:
nit: keep compact, following the pattern of the existing code:
```suggestion
              SizedBox(width: 16,
                child: star),
```